### ISSUE TITLE
Upgrade from CLI not working, class Datas not found (1.7.6.1 to 1.7.6.5)

### ISSUE THREAD

**Issue by manoaratefy** [PrestaShop/PrestaShop#19557]
When I run php install/upgrade/upgrade.php on CLI, it doesn't work. It shows:

```
PHP Fatal error:  Uncaught Error: Class 'Datas' not found in /www/install/init.php:73
Stack trace:
#0 /www/install/upgrade/upgrade.php(52): require_once()
#1 {main}
  thrown in /www/install/init.php on line 73
```

Opening http://www.mysite.com/install/upgrade/upgrade.php on browser doesn't have this issue.

#### Additional information

* PrestaShop version: 1.7.6.1, upgrading to 1.7.6.5
* PHP version: PHP 7.2.31-1+0~20200514.41+debian8~1.gbpe2a56b

**Comment by florine2623** [comment]
Hi @manoaratefy ,

Have you followed this documentation ?
https://devdocs.prestashop.com/1.7/basics/keeping-up-to-date/upgrade/
If yes, could you tell me which steps are problematic ?
Or what steps I should follow, so I can try to reproduce your issue.

Thanks!

**Comment by manoaratefy** [comment]
Hi,

Yes, I followed this documentation. The upgrade is blocked on the step "Database upgrade", when I run `php install/upgrade/upgrade.php` on the webserver.

However, if I choose to run the upgrade on the web browser (using `http://<shop_domain>/install/upgrade/upgrade.php`) the issue is not happening.

**Comment by NeOMakinG** [comment]
Hello @manoaratefy ,

I managed to reproduce it :

![image](https://user-images.githubusercontent.com/14963751/84036312-5a68d200-a99d-11ea-8304-0cfe9f1574f3.png)

I’ll add this to the debug roadmap so that it’s fixed. If you have already fixed it on your end or if you think you can do it, please do send us a pull request!

Thanks!

**Comment by marionf** [comment]
@NeOMakinG could you please add the label "bug" if it's one ?

**Comment by matks** [comment]
@atomiix I think this bug is fixed by https://github.com/PrestaShop/PrestaShop/pull/19010 right ?

**Comment by atomiix** [comment]
@matks yep!

**Comment by atomiix** [comment]
Duplicate of #18723
